justfile

```diff
@@ -40,7 +41,9 @@ install target='release':
  @if (-NOT('{{target}}' -EQ 'debug') -AND -NOT('{{target}}' -EQ 'release')) { \
    Write-Host "target can only be 'debug' or 'release' (default to 'release')"; exit 1; \
  }
- @if ('{{target}}' -EQ 'debug') { cargo +nightly build --example example --features="macros"; } else { cargo +nightly build --example example --features="macros" --release; }
+ cd '{{example_dir}}'; \
+ if ('{{target}}' -EQ 'debug') { cargo +nightly build; } else { cargo +nightly build --release; } \
```

Comment:
the final artifact will now be under a different path:
`red4ext-rs\example\target\release\example.dll`
I think you'll need to update the copy paths too

Reply:
you're completely right, I tried to rush and thought I addressed it while I clearly forgot.
I fixed it in cbd8a14, also simplified the logic to avoid having to `cd ..` here and there.
I also applied the same change in subsequent 1266802 to the CI `build` job, to make sure `example` gets compiled properly.